pkg/chaincode/approve_test.go

```diff
@@ -68,14 +68,15 @@ func NewCommitStatusResponse(result peer.TxValidationCode, blockNumber uint64) *
 
 var _ = Describe("Approve", func() {
 	var channelName string
-	var chaincodeDefinition *chaincode.Definition
+	var chaincodeDefinition *chaincode.CCDefine
```

Comment:
if we compare `Definition` with `CCDefine`, it seems that `CCDefine` adding `ChannelID                string`, `InputTxID                string`, `InitRequired             bool` ,`CollectionConfigPackage  *peer.CollectionConfigPackage` . Hence, I suppose logically, we can use `CCDefine` to replace `Definition` ? or just use a single `Definition struct`.
Otherwise, we create two struct? as CCDefine extend from Definition?
```
type Definition struct {
	Name                 string
	Version              string
	Sequence             int64
	EndorsementPolicy    string
	ValidationPlugin     string
	ValidationParameters []byte
	PackageID            string
}
```

```
type CCDefine struct {
	ChannelID                string
	InputTxID                string
	PackageID                string
	Name                     string
	Version                  string
	EndorsementPlugin        string
	ValidationPlugin         string
	Sequence                 int64
	ValidationParameterBytes []byte
	InitRequired             bool
	CollectionConfigPackage  *peer.CollectionConfigPackage
}
```

Reply:
- `InputTxID` is never used.
- `ChannelID` would probably be better named `ChannelName`.
- `CollectionConfigPackage` is probably better named `Collections` - it is simpler and matches the name used in the Fabric messages.